src/test/java/ruiz/ToDosTest.java

```diff
@@ -0,0 +1,26 @@
+package ruiz;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import ruiz.task.ToDos;
+
+public class ToDosTest {
+    @Test
+    public void saveToString_success() {
```

Comment:
I'm not really sure what this test is supposed to test, perhaps it could be changed to something like testMarkTodoSuccess?